docs/2.9.0/docs/canton/usermanual/manage_participants.rst

```diff
@@ -8,13 +8,15 @@ Manage Participants
 
 .. _participant_bootstrapping:
 
-Manual vs Automatic Initialization
-----------------------------------
+Manual vs Automatic Identity Initialization
+-------------------------------------------
 
 Similarly to a domain, a participant node can either be initialized automatically, the default behaviour,
-or manually. Managing the initialization manually is particularly useful when a participant
-does not control its own identity, when we do not want to store its identity key on the node for security
-reasons, or when we want to set a participant's keys by ourselves.
+or manually. Managing the initialization manually is particularly useful when a participant:
+
+1. Does not control its own identity;
+2. When we do not want to store its identity key on the node for security reasons;
```

Comment:
```suggestion
2. When you do not want to store its identity key on the node for security reasons;
```
According to the style guide, using "we" and "one" as subjects is discouraged. Instead, "you" should be used.